src/app/components/listado-aspirantes/listado_aspirantes/listado_aspirante.component.ts

```diff
@@ -429,7 +429,7 @@ export class ListadoAspiranteComponent implements OnInit, OnChanges {
                 .subscribe((response: any) => {
                     if (response != null && response.Status != '404'
                         && Object.keys(response[0]).length > 0) {
-                        let dataTel = JSON.parse(response[0].Dato)
+                        let dataTel = JSON.parse(response[0].Data)
```

Comment:
Es Dato, este es de la tabla info_complementaria_tercero
```suggestion
                        let dataTel = JSON.parse(response[0].Dato)
```